test/test_other.py

```diff
@@ -13390,6 +13390,85 @@ def test_windows_batch_file_dp0_expansion_bug(self):
     create_file('build_with_quotes.bat',  f'@"emcc" {test_file("hello_world.c")}')
     self.run_process(['build_with_quotes.bat'])
 
+  @parameterized({
+    'wasm2js': (True,),
+    '': (False,)
+  })
+  def test_add_js_function(self, wasm2js):
+    self.set_setting('INVOKE_RUN', 0)
+    self.set_setting('WASM_ASYNC_COMPILATION', 0)
+    self.set_setting('RESERVED_FUNCTION_POINTERS')
+    self.set_setting('EXPORTED_RUNTIME_METHODS', ['callMain'])
+    if wasm2js:
+      self.set_setting("WASM", 0)
+    src = test_file('interop/test_add_function.cpp')
+    post_js = test_file('interop/test_add_function_post.js')
+    self.emcc_args += ['--post-js', post_js]
+
+    print('basics')
+    self.do_run_in_out_file_test('interop/test_add_function.cpp')
+
+    print('with RESERVED_FUNCTION_POINTERS=0')
+    self.set_setting('RESERVED_FUNCTION_POINTERS', 0)
```

Comment:
These lines were recently changed to ALLOW_TABLE_GROWTH in their original location.  Can you check/update this copy?

Reply:
I must have incorrectly merged main.

Reply:
Ah it's correct on my local copy I just haven't pushed for some reason.